Simulator/ability.py

```diff
@@ -431,7 +426,7 @@ def cassiopeia(champion):
     already_targeted = []
     for c in cone:
         coords = field.coordinates
-        if c[0] >= 0 and c[0] <= 7 and c[1] >= 0 and c[1] <= 6:
+        if 7 >= c[0] >= 0 <= c[1] <= 6:

```

Comment:
This one looks weird tho, I guess it works, but at least separate the 2 conditionals